### PR TITLE
[STINS-006] link-backend-with-mongodb

### PR DESCRIPTION
**Pull request by aidandayvyd**
Issue: Keeping the server functions in one file, linking backend.
            Had trouble installing mongodb and couldn't test, but all fixed now.
	Solution: Copied over code from exemplat and produced console output "App server listening on port 3001!"
	Risk: I don't think there is any risk but I may be uninformed
	Reviewed by: Donghyun, Emily

### CODE REVIEW

**Comment by Alex-Beep-Cao** [comment]
The branch set up haven't follow the instruction. Please check the google doc :
 https://docs.google.com/document/d/1NHMXon2qmlgCULkxZNK789E1JvYhyGnf1Cx6T5pztMo/edit

**Comment by aidandayvyd** [comment]
Branch name should match conventions now, and the other branch should be deleted.  Is this correct?

**Review by dlee658 — APPROVED** [review]
LGTM!

**Review by butterplup — APPROVED** [review]
LGTM. I think.

**Review by lucas2005gao — APPROVED** [review]
LGTM!

File: backend/src/server.js [hunk 1]
```diff
@@ -15,5 +16,5 @@ app.use(express.json());
 app.get('/',(req,res) => {
     res.status(200).json("hello");
 })
-
-app.listen(port, ()=> console.log(`App server listening on port ${port}!`));
\ No newline at end of file
+mongoose. connect('mongodb://localhost:27017/stins', {useNewUrlParser: true})
```
Comment (jekb2019) [inline backend/src/server.js:19]: Are we using mongoDB locally? Or are we using Atlas?

**Review by jekb2019 — CHANGES_REQUESTED** [review]
Are we using Atlas or local mongodb?

File: backend/src/server.js (hunk 1 above)
Comment (dlee658) [reply backend/src/server.js:19]: This was for local, the next story is connecting with MongoDB Atlas

Comment (jekb2019) [reply backend/src/server.js:19]: sweet

**Review by jekb2019 — APPROVED** [review]

**Pull request merged**